package.json

```diff
@@ -26,22 +26,22 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "showdown": "0.x.x"
+    "showdown": "1.0.2"
   },
   "devDependencies": {
-    "chai": "^1.10.0",
+    "chai": "^3.0.0",
```

Comment:
I've checked the Chai changelog and seems that no breaking changes affecting our tests have been introduced in current major version, so It's ok to update.

About the other packages, the "^" operator should do the trick by automatically fetching the latest compatible version. So, in case of non breaking minor version updates, there is no need to update.